### PR TITLE
SASS not working now, using raw CSS

### PR DESCRIPTION
**Pull request by Fernan2**
Same happened on
https://github.com/Soluciones/news_tematica/pull/64

### CODE REVIEW

**Comment by maxvlc** [comment]
:+1:

File: app/views/news_tematica/news_tematicas/show.html.haml [hunk 1]
```diff
@@ -1,9 +1,10 @@
-:sass
-  .cuentaclics
-    background: #FF3
-    color: black
-    padding: 4px
-    margin-right: 8px
+:css
+  .cuentaclics {
+    background: #FF3;
```
Comment (Dygerydoo) [inline app/views/news_tematica/news_tematicas/show.html.haml]: Colors in css should be 'lowercase' `#ff3`, if you want you can leave it, it's not a big problem.

**Comment by Dygerydoo** [comment]
:+1:

File: app/views/news_tematica/news_tematicas/show.html.haml (hunk 1 above)
Comment (Fernan2) [inline app/views/news_tematica/news_tematicas/show.html.haml]: I had no idea! it's the first time I read about this...

Comment (Fernan2) [inline app/views/news_tematica/news_tematicas/show.html.haml]: Just googled and read 2 good reasons for this:

- Lowercase is easier to read for humans. EVERY TEXT IN UPPERCASE TAKES ADDITIONAL BRAINPOWER TO READ.

- The only app that ever produced uppercase hex colors is PhotoShop -> If PhotoShop does it, it should be really bad  ;-P

**Pull request merged**